src/primitive/rectangular_plane.rs

```diff
@@ -0,0 +1,151 @@
+//! Rectangular plane primitive
```

Comment:
I find the term slightly confusing. A plane is infinite, so it can't have a shape like rectangle... Maybe `RectangleOnPlane`? Nothing good comes to mind

Reply:
Yeah, I'm not super happy with the name either but couldn't come up with a better one :/

Reply:
If we rename Rectangle to Rectangle2, we can name this Rectangle3